crates/orchestrator/src/database/mongodb/mod.rs

```diff
@@ -40,6 +40,10 @@ impl MongoDb {
         MongoDb { client }
     }
 
+    pub fn client(&self) -> Client {
+        self.client.clone()
```

Comment:
Nitpick: when I see the call to `clone()`, my first reflex is to check the cost of this operation. I looked at the docs of MongoDB and saw that `Client` uses an `Arc` internally, so this is fine. But it is worth adding a comment explaining why this is fine, because for non Arc-based types doing

```
pub fn client(&self) -> &Client {
  &self.client
}
```

and then letting the caller decide if he wants to clone is better.

TL;DR, add a comment to explain that `Client` uses `Arc` and therefore is cheap to clone.

Reply:
Valid and a great catch, 
added docs to describe that `client` uses `Arc`.